tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/recent_concept_suppression_test.py

```diff
@@ -0,0 +1,216 @@
+
+        INSERT_OBSERVATIONS_QUERY = JINJA_ENV.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.observation`
+                (observation_id, person_id, observation_concept_id, observation_source_concept_id, observation_date,
+                observation_type_concept_id)
+            VALUES
+                -- Not suppressed: Is PPI valid_start_date < cutoff_date - 1 year --
+                (1, 101, 1310093, 0, date('2020-05-05'), 1),
+                (2, 115, 765936, 0, date('2020-05-05'), 2),
+
+                -- Not suppressed: valid_start_date < cutoff_date - 1 year --
+                (3, 116, 37310268, 98, date('2020-05-05'), 1),
+                (4, 116, 0, 37310268, date('2020-05-05'), 1),
+
+                -- Suppressed: Non-PPI and valid_start_date >= cutoff_date - 1 year --
+                (5, 102, 0, 1615361, date('2020-05-05'), 2),
+                (6, 103, 36661764, 0, date('2020-05-05'), 3),
+
+                -- Suppressed:  Default valid_start_date and first usage in less than 12 months --
+                (7, 103, 123, 0, '{{recent_date}}', 0),
+
+                -- Not Suppressed:  Default valid_start_date and first usage more than 12 months ago --
+                (8, 103, 456, 0, '{{old_date}}', 0)
+
+        """).render(fq_dataset_name=self.fq_dataset_name,
+                    recent_date=self.recent_date_str,
+                    old_date=self.old_date_str)
+
+        INSERT_DRUG_EXPOSURES_QUERY = JINJA_ENV.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.drug_exposure`
+                (drug_exposure_id, person_id, drug_concept_id, drug_source_concept_id, drug_exposure_start_date, drug_exposure_start_datetime, drug_type_concept_id)
+            VALUES
+                -- Not suppressed: Is PPI valid_start_date < cutoff_date - 1 year --
+                (1, 101, 1310093, 0, date('2020-05-05'), timestamp('2020-05-05 00:00:00 UTC'), 1),
+                (2, 115, 765936, 0, date('2020-05-05'), timestamp('2020-05-05'), 2),
+
+                -- Not suppressed: valid_start_date < cutoff_date - 1 year --
+                (3, 116, 37310268, 98, date('2020-05-05'), timestamp('2020-05-05 00:00:00 UTC'), 1),
+                (4, 116, 0, 37310268, date('2020-05-05'), timestamp('2020-05-05 00:00:00 UTC'), 1),
+
+                -- Suppressed: Non-PPI and valid_start_date >= cutoff_date - 1 year --
+                (5, 102, 0, 1615361, date('2020-05-05'), timestamp('2020-05-05 00:00:00 UTC'), 2),
+                (6, 103, 36661764, 0, date('2020-05-05'), timestamp('2020-05-05 00:00:00 UTC'), 3),
+
+                -- Suppressed:  Default valid_start_date and first usage in less than 12 months --
+                (7, 103, 123, 0, '{{recent_date}}', timestamp('2020-05-05 00:00:00 UTC'), 0),
+
+                -- Not Suppressed:  Default valid_start_date and first usage more than 12 months ago --
+                (8, 103, 456, 0, '{{old_date}}', timestamp('2020-05-05 00:00:00 UTC'), 0)
```

Comment:
Nice Commenting!  This will definitely help whoever comes back to update and maintain the code.